core/system/rule_manager.go

```diff
@@ -55,12 +34,22 @@ func GetRules() []*SystemRule {
 }
 
 // LoadRules loads given system rules to the rule manager, while all previous rules will be replaced.
-func LoadRules(rules []*SystemRule) (bool, error) {
```

Comment:
The code related to system rule manager is duplicated  with https://github.com/alibaba/sentinel-golang/pull/72.
I will review flow rule related code.
It's better to remove system rule related code.
Thanks

Reply:
done